src/types.md

```diff
@@ -400,6 +471,33 @@ let bo: Binop = add;
 x = bo(5,7);
 ```
 
+### Other traits
+
+Closure types implement the following traits, if allowed to do so by the
+captured values:
+
+* `[Sized]`
+* `[Send]`
+* `[Sync]`
+* `[Clone]`
```

Comment:
Given how Clone and Copy are lang items and Send and Sync aren't (I think? Maybe one of them is?), I'd rather see Clone and Copy listed first.

Reply:
`Sync` is, `Send` isn't. I think it would look better reordered though, so I will do that.